modules/UI/side_pannels/settings/SettingsMenu.js

```diff
@@ -1,10 +1,65 @@
-/* global $, APP, AJS, interfaceConfig, JitsiMeetJS */
+/* global $, _, APP, AJS, interfaceConfig, JitsiMeetJS */
 
 import UIUtil from "../../util/UIUtil";
 import UIEvents from "../../../../service/UI/UIEvents";
 import languages from "../../../../service/translation/languages";
 import Settings from '../../../settings/Settings';
 
+const sidePannelsContainerId = 'sideToolbarContainer';
+const compiledTpl = _.template(`
+    <div id="settings_container" class="sideToolbarContainer__inner">
+        <div class="title" data-i18n="settings.title"></div>
+        <form class="aui">
+            <div id="languagesSelectWrapper" class="first hide">
```

Comment:
It should be `<div id="languagesSelectWrapper" class="sideToolbarBlock first hide">`